docs/specviz/displaying.rst

```diff
@@ -29,7 +29,68 @@ Data can be selected and de-selected by clicking on the "hammer and screwdriver"
 Pan/Zoom
 ========
 
-Coming soon
+Included with the Specviz tool are multiple manners to change your view/perspective on the data displayed, depending on your environment:
+
+Interactive Pan/Zoom (Desktop App interface)
+--------------------------------------------
+You can find the following Pan/Zoom tools available in the tool drawer of the viewer:
+
+2D Bidirectional Pan/Zoom
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The 2D Pan/Zoom Icon allows you to zoom using the scroll wheel. The window will zoom into the area around your cursor:
+
+.. image:: ../img/SpecViz_2D_Zoom.gif
+    :alt: Zooming in Specviz with the Scroll Wheel
+
+To pan, simply click and drag the window
+
+Horizontal/Vertical Zoom
+^^^^^^^^^^^^^^^^^^^^^^^^
+The Horizontal and Vertical Zoom tools allow you to zoom along each axis, while locking the other. Zoom by scrolling
+
+.. note::
+    A gif here showing how these tools manipulate the view, similar to above may be useful here
+
+
+API Pan/Zoom (Notebook interface)
+---------------------------------
+The Specviz helper contains a set of convinence methods to programmatically define the view of the spectrum viewer.
+
+For information about the Specviz, and other, helpers, and how to instantiate one, please see `helpers <https://readthedocs.org/404>`_
+
+.. warning::
+    The helper page has not yet been written/defined. Until that documentation is written, you may instantiate a Specviz Helper via:
+    ::
```

Comment:
```suggestion
You may instantiate a Specviz Helper via:
```
Again, this note should be internal and should not be on the user facing documentation.